### PR TITLE
Fix Release version text 

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

~Since the site is launched by the following command,~
~the `RELEASE` file should start with `tag: vX.Y.Z`.~
~https://github.com/pipe-cd/pipecd/blob/daf0a09568c7333b8be11ad38988441eafe2b842/Makefile#L153~

Now:  `Generated` is shown. (other docs too)
image~

**Which issue(s) this PR fixes**:

N/A

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

**Comment by khanhtc1202** [comment]
@t-kikuc could you test with make release command?

**Review by ffjlabo — APPROVED** [review]
🚀

**Comment by t-kikuc** [comment]
@khanhtc1202  I'm testing and fixing

**Comment by t-kikuc** [comment]
@khanhtc1202 I fixed [9e4cab5](https://github.com/pipe-cd/pipecd/pull/5192/commits/9e4cab5c753294eaf0dced11eebbce9e8010dfe0)

File: RELEASE [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Generated by `make release` command.
-# DO NOT EDIT.
-tag: v0.48.8
+tag: v0.48.8 # Generated by `make release` command. DO NOT EDIT.
```
[the remark this answers is not in the document]
Comment (t-kikuc) [reply RELEASE]: @khanhtc1202  fixed
[9612286](https://github.com/pipe-cd/pipecd/pull/5192/commits/9612286e8ed9c7cfa48d5abb8b648c42b7a6e30f)

**Review by khanhtc1202 — APPROVED** [review]
Nice 👍

**Review by ffjlabo — APPROVED** [review]
Thank you!

**Pull request merged**